packages/core/package.json

```
@@ -217,7 +217,7 @@
     "@emotion/weak-memoize": "^0.3.0",
     "@graphql-ts/extend": "^1.0.0",
     "@graphql-ts/schema": "^0.6.0",
-    "@graphql-typed-document-node/core": "3.1.1",
+    "@graphql-typed-document-node/core": "3.1.2",
```

Why isn't this a caret :thinking: , @borisno2 (#7878)

---

I don't recall a reason why, I'll update...
```suggestion
    "@graphql-typed-document-node/core": "^3.1.2",
```